Add installation requirements for Python and error to FAQ. (#2156)
Percona-Lab/pmm-submodules: README.md

This repository serves the purpose of creating and/or updating the feature builds for PMM Server and PMM Client. It is auxiliary
to our build system managed by Jenkins as it helps pull the right branches from different repositories that PMM [consists of](https://github.com/percona/pmm).

# Installation dependencies

> If you build with Python's script first of all install requirements:

Reviewer: ```suggestion
If you build with Python's script then you need install requirements:
```
